cosmos/operators/docker.py

```diff
@@ -33,23 +33,23 @@ class DbtDockerBaseOperator(DockerOperator, DbtBaseOperator):
     def __init__(
         self,
         image: str,  # Make image a required argument since it's required by DockerOperator
-        **kwargs,
+        **kwargs: Any,
     ) -> None:
         super().__init__(image=image, **kwargs)
 
-    def build_and_run_cmd(self, context: Context, cmd_flags: list[str] | None = None):
+    def build_and_run_cmd(self, context: Context, cmd_flags: list[str] | None = None) -> Any:
         self.build_command(cmd_flags, context)
-        self.log.info(f"Running command: {self.command}")
+        self.log.info(f"Running command: {self.command}")  # type: ignore[has-type]
```

Comment:
Would there be an alternative to ignore this error?

Reply:
Resolving in a separate, future ticket.